account_menu_invoice_refund/views/account_invoice_view.xml

```diff
@@ -43,8 +43,8 @@
         <field name="res_model">account.move</field>
         <field name="view_mode">tree,form</field>
         <field eval="False" name="view_id" />
-        <field name="domain">[('type', 'in', ('in_invoice', 'in_refund'))]</field>
-        <field name="context">{'type':'in_invoice', 'journal_type': 'purchase'}</field>
+        <field name="domain">[('move_type', 'in', ('in_invoice', 'in_refund'))]</field>
+        <field name="context">{'move_type':'in_invoice', 'journal_type': 'purchase'}</field>
```

Comment:
change move_type to default_move_type. i think that journal_type is not required